src/cross-validate.js

```diff
@@ -116,7 +116,7 @@ export default class CrossValidate {
       let dclone = data.slice(0);
       let testSet = dclone.splice(i * size, size);
       let trainSet = dclone;
-      let result = this.testPartition(options, trainOpts, trainSet, testSet);
+      let result = this.testPartition(this.options, trainOpts, trainSet, testSet);
```

Comment:
Nice find!  We could (should) simplify this.  We don't need to pass options to `testPartition` because it is part of `this`, can you remove here and reference there?

Reply:
I take it back, `this.options` just needs removed in this case. I'll go ahead and commandeer this as I need it for work this morning.